pom.xml

```diff
@@ -80,16 +80,10 @@
             <version>42.2.5</version>
         </dependency>
 
-        <dependency>
-            <groupId>org.eclipse.persistence</groupId>
-            <artifactId>eclipselink</artifactId>
-            <version>${eclipselink.version}</version>
-        </dependency>
         <dependency>
             <groupId>org.eclipse.persistence</groupId>
             <artifactId>org.eclipse.persistence.jpa</artifactId>
             <version>${eclipselink.version}</version>
-            <scope>compile</scope>
```

Comment:
The default `scope` is `compile` so this doesn’t make any difference (we usually don’t include `compile` in our Maven dependencies but I think we do for EclipseLink because it’s in their documentation)

Reply:
Is it worth reinstating it? Sounds like it's surplus to requirements